docs/contents/news/1-19/1/CHANGELOG-1.19.1.md

```diff
@@ -0,0 +1,53 @@
+# Changelog for v.1.19.1
+
+This changelog highlights the changes from EKS-D [v.1.18.1](https://github.com/aws/eks-distro/tree/v1-18-eks-1) to 
```

Comment:
nit: no dot after v in `v1.18.1`. Same for `v1.19.1`.

Reply:
Good catch

Reply:
Is this something that we want to highlight, because once we start cutting releases at the same time like `1-18-3` with `1-19-2`, do we compare 1-19 with the previous 1-19 changes or the 1-18 changes as well?

Reply:
Instead did we want to give a high level overview here on what we took from 1.19.<whatever of k8s 1.19 we are taking> and talk about the versions of components as well as patches we introduced? If we are comparing it to our version of 1.18 it might be confusing, but at the same time it might be beneficial idk. Definitely would like to get thoughts on this.

Reply:
Yeah I considered this too. I re-phrased some of this in revision 2 of this PR. However, I think it's worth highlighting why this release dropped some patches that were in 1.18.1. I'm not sure the best way to phrase this for when we start cutting 1-18-whatever, ect. releases